label_studio/ml_model_providers/models.py

```diff
@@ -40,8 +40,11 @@ class ModelProviderConnectionScopes(models.TextChoices):
 
     updated_at = models.DateTimeField(_('updated at'), auto_now=True)
 
+    # Check if user is Admin or Owner
+    # This will need to be updated if we ever use this model in LSO as `is_owner` and
+    # `is_administrator` only exist in LSE
     def has_permission(self, user):
-        return user.active_organization == self.organization
+        return (user.is_administrator or user.is_owner) and user.active_organization_id == self.organization_id
```

Comment:
can we add a test case where user is not admin, or not owner and fails to create or modify model provider

Reply:
saw test in lse